packages/account/src/Sections/Security/TwoFactorAuthentication/two-factor-authentication.jsx

```diff
@@ -167,7 +176,9 @@ class TwoFactorAuthentication extends React.Component {
                                                         )}
                                                     </h4>
                                                     <div className='two-factor__qr--code'>
-                                                        <span>{secret_key}</span>
+                                                        <Text size='xs' text_align='left'>
```

Comment:
Same here

Reply:
I answered above, Are you agree?

Reply:
I mean `text-align` property not whole thing. Sorry for bad highlighting :))